src/api/app/assets/stylesheets/webui/new_watchlist/watchlist.scss

```
@@ -0,0 +1,4 @@
+.color-inverted-svg {
```

:nitpick:

Is this really only for SVGs? This works for any image. I suggest renaming the class to:

```suggestion
.invert-color {
```